packages/svelte/tests/runtime-runes/samples/snippet-optional-arguments-defaults/_config.js

```diff
@@ -0,0 +1,7 @@
+import { flushSync } from 'svelte';
+import { test } from '../../test';
+
+export default test({
+	html: '012',
+	ssrHtml: '012'
```

Comment:
we don't need both, unless they're different

```suggestion
	html: '013'
```